### ISSUE TITLE
[FEAT] Remove unused mods

### ISSUE THREAD

**Issue by murkl** [aclist/dztui#162]
An option that deletes all mods from Steam that are no longer needed would be super helpful. So that all mods that are not used by servers from 'My saved servers' are deleted. The reason for this is the large amount of data that some of the mods consume.

**Comment by aclist** [comment]
As a preliminary step, I've added bulk mod deletion to the Testing branch. This feature is accessible via `Options > List installed mods`. It does not currently show you which mods are "stale", but you can bulk select/unselect mods and use `Ctrl+click` to add/delete selections before triggering deletion. This won't work correctly with Manual Mod install mode for the reasons mentioned above.

**Comment by aclist** [comment]
This feature has now been added to the `List installed mods` table on the `Testing` branch (`5.6.0-beta.5`). Clicking `Highlight stale` will search through servers **in your saved servers list** and compare their modlists against the mods you **have installed locally**. If you have mods installed that aren't used on any of those servers, they will be marked in red. You can use this information if you choose to delete them. To unmark these mods, click `Unhighlight stale`.

**Comment by aclist** [comment]
Thank you. Let me know if you run into difficulties. This ticket will be formally closed once this feature is merged into the stable branch with the upcoming 5.6.0 release.

**Comment by aclist** [comment]
This should be fully working and integrated with the latest version live on both stable and testing branches now.
